components/eam/src/physics/cam/zm_conv.F90

```diff
@@ -1159,7 +2083,7 @@ subroutine convtran(lchnk   , &
    integer, intent(in) :: nstep             ! Time step index
 
    real(r8), intent(in) :: dpdry(pcols,pver)       ! Delta pressure between interfaces
-
+   real(r8), intent(in) :: dt               ! 2 delta t (model time increment)
```

Comment:
Are you sure this is twice the physics time step?

Reply:
This is not related to this PR or ZM enhancement. It has been there since NCAR CCM3 (~20 years ago). It is still true for Eul dynamics but not for SE dynamics. There are a lot of similar statement (2 delta t) in the E3SMv2. For now, I would suggest keeping it as is.